featuros-plugin/src/main/java/io/fourfinanceit/featuros/Featuros.java

```diff
@@ -0,0 +1,66 @@
+package io.fourfinanceit.featuros;
+
+import jodd.http.HttpException;
+import jodd.http.HttpResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+import java.util.Optional;
+
+import static jodd.http.HttpRequest.post;
+
+/**
+ * Plugin that reaches out to featuros dashboard and notifies of current deployment.
+ * When not using a boot-starter module, it has to be called manually, e.g.:
+ * <pre>
+ *     public static void main(String... args) {
+ *         SpringApplication.run(args);
+ *         Featuros.notification();
+ *     }
+ * </pre>
+ */
+public class Featuros {
+
+    private static final Logger log = LoggerFactory.getLogger(Featuros.class);
```

Comment:
I understand you Jakub, but we're not throwing in groovy or lombok because there's no real need for it. Sure it would save a line there and there, however it does not align with a lightweight library approach we have taken. This plugin is a library, not a framework, not an end product. Let's close the discussion because it doesn't lead anywhere as I see it.

Sent from my iPhone

> On 01 Jul 2015, at 12:38, Jakub Nabrdalik <notifications@github.com> wrote:
> 
> In featuros-plugin/src/main/java/io/fourfinanceit/featuros/Featuros.java:
> 
> > +
> > +import static jodd.http.HttpRequest.post;
> > +
> > +/**
> > + * Plugin that reaches out to featuros dashboard and notifies of current deployment.
> > + * When not using a boot-starter module, it has to be called manually, e.g.:
> > + * <pre>
> > + *     public static void main(String... args) {
> > + *         SpringApplication.run(args);
> > + *         Featuros.notification();
> > + *     }
> > + * </pre>
> > + */
> > +public class Featuros {
> > +
> > +    private static final Logger log = LoggerFactory.getLogger(Featuros.class);
> The whole thing is unecessary. Compare with:
> @Slf4j
> 
> —
> Reply to this email directly or view it on GitHub.
>